src/parser/test/ParserTest.cpp

```diff
@@ -948,6 +948,12 @@ TEST(Parser, AdminOperation) {
         auto result = parser.parse(query);
         ASSERT_TRUE(result.ok()) << result.status();
     }
+    {
+        GQLParser parser;
+        std::string query = "SHOW PARTS 666";
```

Comment:
If you're showing one part, it's better to use `SHOW PART 666`.

Reply:
I'll inform you after merged, you can hold on doc.